cpp/tests/large_strings/parquet_tests.cpp

```diff
@@ -69,3 +70,57 @@ TEST_F(ParquetStringsTest, ReadLargeStrings)
   // go back to normal threshold
   unsetenv("LIBCUDF_LARGE_STRINGS_THRESHOLD");
 }
+
+TEST_F(ParquetStringsTest, ChunkedReadLargeStrings)
+{
+  // Construct a table with one large strings column > 2GB
+  auto const wide = this->wide_column();
+  auto input      = cudf::concatenate(std::vector<cudf::column_view>(120000, wide));  ///< 230MB
+
+  int constexpr multiplier = 12;
+  std::vector<cudf::column_view> input_cols(multiplier, input->view());
+  auto col0 = cudf::concatenate(input_cols);  ///< 2.70GB
+
+  // Expected table
+  auto const expected    = cudf::table_view{{col0->view()}};
+  auto expected_metadata = cudf::io::table_input_metadata{expected};
+  expected_metadata.column_metadata[0].set_encoding(
+    cudf::io::column_encoding::DELTA_LENGTH_BYTE_ARRAY);
+
+  // Write to Parquet
+  auto const filepath = g_temp_env->get_temp_filepath("ChunkedReadLargeStrings.parquet");
+  cudf::io::parquet_writer_options out_opts =
+    cudf::io::parquet_writer_options::builder(cudf::io::sink_info{filepath}, expected)
+      .compression(cudf::io::compression_type::ZSTD)
```

Comment:
why ZSTD?
also, is it any faster to write to a `std::vector` instead of a file?

Reply:
^ Same as above. Probably, the test takes 2s to run anyway, 5s with `compute-sanitizer`

Reply:
I think we should prefer the defaults if there's no reason to do otherwise.

Well... 2s is still hefty for a single test. I do suspect removing IO does not help, so not pushing for it here.

Reply:
All done. removed!

Reply:
Discussed offline: We have disabled this test by default as it is too brittle and depends on empirical pass_read_limit, ZSTD scratch space, and DLBA encoding which may change in the future leading to false failures.